libp2p/Host.cpp

```diff
@@ -672,37 +675,39 @@ void Host::run(boost::system::error_code const& _ec)
     if (!m_run || _ec)
         return;
 
-    if (auto nodeTable = this->nodeTable()) // This again requires x_nodeTable, which is why an additional variable nodeTable is used.
+    // This again requires x_nodeTable, which is why an additional variable nodeTable is used.
+    if (auto nodeTable = this->nodeTable())
         nodeTable->processEvents();
 
     // cleanup zombies
     DEV_GUARDED(x_connecting)
         m_connecting.remove_if([](weak_ptr<RLPXHandshake> h){ return h.expired(); });
-    DEV_GUARDED(x_timers)
-    m_timers.remove_if([](unique_ptr<io::deadline_timer> const& t) {
+    DEV_GUARDED(x_networkTimers)
+    m_networkTimers.remove_if([](unique_ptr<io::deadline_timer> const& t) {
```

Comment:
Enclose with `{}`

Reply:
Ah, the clang-format messed things up. Will add braces.